static/templates/single_message.handlebars

```diff
@@ -15,6 +15,9 @@
 
             {{#unless status_message}}
               {{msg/sender_full_name}}
+              {{#if is_bot}}
+                <img class="bot-icon" src="/static/images/bot.png" />
+              {{/if}}
```

Comment:
In theory we should have similar logic in the next case for handling bot messages starting with `/me`.